### PR TITLE
Add image into dashboard tile interface

### PR DESCRIPTION
**Pull request by cdjackson**
As discussed, this adds the image link into the dashboard tile interface. I've updated the io.rest and classic ui bundles, but not other bundles that may use this interface (eg paperui).
Note that for the classic ui I left the image in the existing place in the dashboard bundle since I couldn't immediately see where the classic ui was now serving up its data from (I guess this is an extension of a bundle in ESH maybe?).
Signed-off-by: Chris Jackson <chris@cd-jackson.com>

### CODE REVIEW

**Comment by maggu2810** [comment]
> since I couldn't immediately see where the classic ui was now serving up its data from (I guess this is an extension of a bundle in ESH maybe?).

You are right: https://github.com/eclipse/smarthome/tree/master/bundles/ui/org.eclipse.smarthome.ui.classic

File: bundles/ui/org.openhab.ui.dashboard/src/main/java/org/openhab/ui/dashboard/DashboardTile.java [hunk 1]
```diff
@@ -31,6 +31,13 @@
     String getUrl();
     
     /**
+     * The url to point to for the dashboard tile.
+     * (if it is a local UI, it should be a relative path starting with "../")
+     * @return the tile url
+     */
+    String getDashboardTile();
```
Comment (kaikreuzer) [inline bundles/ui/org.openhab.ui.dashboard/src/main/java/org/openhab/ui/dashboard/DashboardTile.java]: May I suggest to rename this method to getImageUrl()? Having a DashboardTile.getDashboardTile() somehow sounds recursive to me ;-)
Apart from that: Thanks for the quick PR! I actually didn't think of such an easy solution, I was somehow wondering how to send the image streams around and serving them through the servlet. But solving this with simple urls is of course much better!

Comment (cdjackson) [inline bundles/ui/org.openhab.ui.dashboard/src/main/java/org/openhab/ui/dashboard/DashboardTile.java]: No probs - I’ve changed the method so hopefully all good to go :)

Cheers
Chris

**Comment by kaikreuzer** [comment]
Thanks, you are really incredibly fast!

**Pull request merged**